loops-and-arrays.js

```diff
@@ -53,3 +63,7 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 24 is 13824',
 //   'Het volume van 11 is 1331'
 // ]
+
+for (let i = 0; i < squares.length; i++) {
+    console.log("Het volume van " + squares[i] + " is " + squares[i] * squares[i] * squares[i]);
```

Comment:
De uitkomst klopt! 🔥

Verder hetzelfde als bij de vorige opdrachten, de array zelf aanpassen. Je kan er verder nog een const volume bij plakken om de zin korter te maken.
Uitwerking geeft dit:

for (let i = 0; i < squares.length; i++) {
  const volume = squares[i] * squares[i] * squares[i];

  squares[i] = "Het volume van " + squares[i] + " is " + volume;
}

console.log(squares);